### PR TITLE
feat: improve payment confirmation mailers

### PR DESCRIPTION
**Pull request by foochifa**
## Problem

Closes FRM-1261

## Solution

**Breaking Changes** 

- [ ] No - this PR is backwards compatible  

## Before & After Screenshots

**BEFORE**:

image

**AFTER**:

image

## Tests

Products
- [ ] Create a payment by products form
- [ ] Submit the form and pay
- [ ] Check your email
- [ ] You should see the confirmation email design as stipulated in the linear issue
- [ ] Check the information in the confirmation email is correct
- [ ] Download the pdf from the email

Variable
- [ ] Create a variable payment form
- [ ] Submit and pay
- [ ] Check your email
- [ ] You should see the confirmation email design as stipulated in the linear issue
- [ ] Check the information in the confirmation email is correct
- [ ] Download the pdf from the email

### CODE REVIEW

File: src/app/views/templates/payment-confirmation.view.html [hunk 1]
```diff
@@ -1,12 +1,124 @@
...
+          font-size: 16px;
+          line-height: 24px;
+          letter-spacing: -0.006em;
+
+          color: #474747;
+
+          margin-bottom: 32px;
+        "
+      >
+        <p style="margin: 0px">Form: <%= formTitle %></p>
```
Comment (wanlingt) [inline src/app/views/templates/payment-confirmation.view.html]: Looking at the figma, it seems like the inputs `formTitle`, `submissionId`, etc are bolded (larger font-weight size)

**Review by wanlingt — APPROVED** [review]
looks good! will leave stacey to review the design

File: src/app/views/templates/payment-confirmation.view.html (hunk 1 above)
Comment (foochifa) [reply src/app/views/templates/payment-confirmation.view.html]: Thanks for catching this @wanlingt have changed it here [a31fff9](https://github.com/opengovsg/FormSG/pull/6704/commits/a31fff9352e0dce9e377d145e79afa8b69dd156b) and refactored the styles!

**Comment by foochifa** [comment]
Note: gmail doesn't support [custom fonts](https://stackoverflow.com/questions/39144407/how-to-use-custom-font-in-gmail) and can only use CSS [web safe fonts](https://www.w3schools.com/cssref/css_websafe_fonts.php)

🥲

Have gotten go ahead from stacey on the current state of the mailer!

**Pull request merged**